### PR TITLE
foundation board: address needs of different user types

### PR DESCRIPTION
**Pull request by fricklerhandwerk**
This proposal was prompted by @MMesch, who brought it up at the [NixCon governance workshop](https://discourse.nixos.org/t/nixcon-governance-workshop/32705/6#line-between-community-and-companies-2) and also gave [a talk on the subject at the 23.11 ZHF workshop](https://discourse.nixos.org/t/zurich-23-11-zhf-hackathon-and-ux-workshop-report/37848#lightning-talks-3), and I agree this should be made explicit.

Asking @nixos/foundation to consider for inclusion/clarification of their responsibilities.

### CODE REVIEW

File: community/teams/foundation-board.tt [hunk 1]
```diff
@@ -38,6 +38,12 @@
     <li>
       Handle sponsoring and donations
     </li>
+    <li>
+      Encourage ontributions by individuals and organisatons alike
```
Comment (henrik-ch) [inline community/teams/foundation-board.tt:42]:
```suggestion
      Encourage contributions by individuals and organisations alike
```

Comment (fricklerhandwerk) [reply community/teams/foundation-board.tt:42]: Thanks! I love typo fixes because it shows someone actually reads that stuff.

**Review by thufschmitt — APPROVED** [review]
Makes sense to me. Can you also mirror that in https://github.com/NixOS/foundation/blob/master/role_and_responsibilities.md ?

**Comment by thufschmitt** [comment]
Merged on the foundation side. Letting a member of @NixOS/marketing-team merge this one

**Pull request merged**